.eslintrc

```diff
@@ -68,6 +67,8 @@
     "space-infix-ops": "error",
     "eol-last": ["error", "always"],
     "comma-spacing": ["error", { "before": false, "after": true }],
+    "semi": "off",
```

Comment:
Waarom standaard semi off?

Reply:
Omdat die typescript linting dat ook doet, maar schijnt anders te kunnen conflicteren

Reply:
Eslint gaat hierdoor wel op z'n nek, kijk maar in de CI

Reply:
Gefixt